File: examples/museum/oseg_generated/java/DeleteSpecialEvent_default_example.java
package org.openapimuseum.client.examples;

import org.openapimuseum.client.ApiException;
import org.openapimuseum.client.Configuration;
import org.openapimuseum.client.api.*;
import org.openapimuseum.client.auth.*;
import org.openapimuseum.client.model.*;

import java.io.File;
import java.util.List;
import java.util.Map;

public class DeleteSpecialEvent_default_example
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        try
        {
            var apiCaller = new EventsApi(config);

            apiCaller.deleteSpecialEvent(
                "dad4bce8-f5cb-4078-a211-995864315e39"
            );
        } catch (ApiException e) {
            System.err.println("Exception when calling Events#deleteSpecialEvent");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: examples/museum/oseg_generated/csharp/DeleteSpecialEvent_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class DeleteSpecialEventDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            new EventsApi(config).DeleteSpecialEvent(
                eventId: "dad4bce8-f5cb-4078-a211-995864315e39"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#DeleteSpecialEvent: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
